### PR TITLE
Symmachus's work №3

### PR DESCRIPTION
**Pull request by Symmachus**
So, I implemented the additions and fixes, which I already told Loup on Discord.

### CODE REVIEW

File: WTWSMS/history/characters/avar.txt [hunk 1]
```diff
@@ -54,21 +54,6 @@
...
-	culture="avar"
-	father=31048 
-	530.1.1={
-		birth="530.1.1"
-	}
-	602.1.1={
-		death="602.1.1"
-	}
-}
-
```
Comment (loup99) [inline WTWSMS/history/characters/avar.txt]: The other Bayan doesn't have any father entry.

Comment (Symmachus) [reply WTWSMS/history/characters/avar.txt]: > The other Bayan doesn't have any father entry.

This deleted Bayan was descended from Slavic rulers, which IMO is very strange

Comment (loup99) [reply WTWSMS/history/characters/avar.txt]: For the Avars? Have you checked what genealogy tree that made?

Comment (Symmachus) [reply WTWSMS/history/characters/avar.txt]: > For the Avars? Have you checked what genealogy tree that made?

There was the two Bayans and i preserved the one who owns Avar khaganate and have a son. Second Bayan was son of the Slavic ruler. So, now the Avar dynasty starts with Bayan I.

Comment (loup99) [reply WTWSMS/history/characters/avar.txt]: Ok.

**Pull request merged**